operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,551 @@
+required to maintain the HA of critical management services because of unplanned outages. This feature ensures that CSM can tolerate failures of racks and
+prevent a system wide outage, where such outage encompasses the inability for completing current jobs or scheduling new ones. This feature doesn't address
+planned downtime of racks. 
+
+A similar problem emerges for utility storage on management storage nodes. Even though these are not orchestrated by `Kubernetes', incorrect physical placement
+or software configuration, specifically around replication, may cause utility storage service disruptions in the event of rack-level failures.
+
+Please note that the Rack Resiliency feature is disabled by default, admin needs to enable during install/ upgrade procedure as explained in the respective section below.
+
+### Key features
+
+* Validates and reports node placement configuration. This allows administrators to reconfigure, if required, in order to ensure that critical management services can tolerate rack failures.
+* Provides tolerance for a single rack failure within the Management Plane Failure Domain (MPFD), ensuring that it does not lead to a system-wide outage.
+* Ensures load balancing and high availability of critical services essential for the seamless execution and completion of user jobs.
+* Monitors and reports failures of management nodes and racks which are part of the MPFD.
+* Monitors and reports critical services health and their balance across racks.
+* Aligns with future plans for similar functionality in next generation systems management solutions
+
+## Key Terminology
+* Placement: Arrangement of Management Nodes across racks. Note that zone awareness primitives in `Kubernetes` refer to this as "topology".
+* Failure Domain: Failure domains are zones which includes infrastructure that provides availability for CSM services.
+* Zone: A zone represents a logical failure domain. It is common for `Kubernetes` clusters to span multiple zones for increased availability. 
+* `Kubernetes` topology zones: You can use topology spread constraints to control how pods are spread across your cluster among failure-domains such
+  as regions, zones, nodes, and other user-defined topology domains. This can help to achieve high availability as well as efficient resource utilization.
+* Ceph: `Ceph` is the utility storage platform that is used to enable pods to store persistent data. It is deployed to provide block, object, and file storage
+  to the management services running on Kubernetes, as well as for telemetry data coming from the compute nodes.
+* CFS: Configuration Framework Service (CFS) is available on systems for remote execution and configuration management of nodes and boot images.
+  CFS configures nodes and images via a GitOps methodology. All configuration content is stored in the Version Control Service (VCS) and is
+  managed by authorized Infrastructure Administrators.
+
+## Critical services for RR

```

Comment:
This section should be moved into its own page. This current page is trying to cover too much.